benchmarks/asv_numpy.conf.json

```diff
@@ -41,15 +45,15 @@
     //"install_timeout": 600,
 
     // the base URL to show a commit for the project.
-    "show_commit_url": "https://github.com/kymatio/kymatio/commit/",
+    //"show_commit_url": "https://github.com/kymatio/kymatio/commit/",
 
     // The Pythons you'd like to test against.  If not provided, defaults
     // to the current version of Python used to run `asv`.
     "pythons": ["3.7"],
 
     // The list of conda channel names to be searched for benchmark
     // dependency packages in the specified order
-    // "conda_channels": ["conda-forge", "defaults"],
+    "conda_channels": ["defaults",],//"conda-forge",
```

Comment:
Again, remove commented-out stuff. It's fine to have comments, but they should be actual comments (that may contain code), not just code.

Reply:
ok